x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/plugin/CanMatchIT.java

```diff
@@ -78,26 +78,41 @@ public void testCanMatch() {
                     handler.messageReceived(request, channel, task);
                 });
             }
-            EsqlQueryResponse resp = run("from events_*", randomPragmas(), new RangeQueryBuilder("@timestamp").gte("2023-01-01"));
-            assertThat(getValuesList(resp), hasSize(4));
-            assertThat(queriedIndices, equalTo(Set.of("events_2023")));
-            queriedIndices.clear();
-
-            resp = run("from events_*", randomPragmas(), new RangeQueryBuilder("@timestamp").lt("2023-01-01"));
-            assertThat(getValuesList(resp), hasSize(3));
-            assertThat(queriedIndices, equalTo(Set.of("events_2022")));
-            queriedIndices.clear();
-
-            resp = run("from events_*", randomPragmas(), new RangeQueryBuilder("@timestamp").gt("2022-01-01").lt("2023-12-31"));
-            assertThat(getValuesList(resp), hasSize(7));
-            assertThat(queriedIndices, equalTo(Set.of("events_2022", "events_2023")));
-            queriedIndices.clear();
-
-            resp = run("from events_*", randomPragmas(), new RangeQueryBuilder("@timestamp").gt("2021-01-01").lt("2021-12-31"));
-            assertThat(getValuesList(resp), hasSize(0));
-            assertThat(queriedIndices, empty());
-            queriedIndices.clear();
+            try (EsqlQueryResponse resp = run("from events_*", randomPragmas(), new RangeQueryBuilder("@timestamp").gte("2023-01-01"))) {
```

Comment:
Hmm.. I updated these tests to close the response, so created a merge conflict here. Sorry.

Reply:
Merge conflicts are life.